gui/src/components/FormNotifications.js

```diff
@@ -2,42 +2,27 @@ import React, { Fragment } from 'react'
 import Flash from './Flash'
 import { Link } from 'react-static'
 
-export const BridgeAndJobNotifications = props => {
-  const { error, success, networkError, authenticated, classes, jobOrBridge } = props
-  return (
-    <Fragment>
-      {error.length > 0 &&
-        authenticated && (
-          <Flash error className={classes.flash}>
-            {(Array.isArray(error) && error.map((msg, i) => <span key={i}>{msg}</span>)) || error}
+export default class FormNotifications extends React.Component {
+  render () {
+    const { success, classes, jobOrBridge } = this.props
+    return (
+      <Fragment>
+        {JSON.stringify(success) !== '{}' && (
+          <Flash success className={classes.flash}>
```

Comment:
Note: Error reporting has been removed from this class.

@rupurt @NavyAdmiral Now that we've placed `create` error reporting at the top level in `Layout.js` and this only notifies successes, I believe we should ultimately move this to `Layout.js` (or top level) as well.

In other words, all notifications, successes and errors, should be at the top level and not embedded in contextual components. No need to do that in this PR but that's where we are moving. It will also get rid of the `jobOrBridge` but we'll have to come up with a way to generate the show link.

Top level error reporting: https://github.com/smartcontractkit/chainlink/pull/537/files#diff-4c5a8365a1292c74b8df56f96414fc09R183

Reply:
@dimroc reporting errors in form components was a design choice I made because I wanted to display Flash notifications within the `<TabContainer/>`. It will be better if we move it to top because the former approach results in complications for other notifications we might need in future outside these forms. Also, `jobOrBridge` was very convoluted and I very much disliked it. :shipit: